### PR TITLE
Set last 4 digits in CardComponentState

### PR DESCRIPTION
**Pull request by jreij**
## Summary
Set the last 4 digits of the card number when the CardComponentState is valid.

### CODE REVIEW

File: card/src/main/java/com/adyen/checkout/card/CardComponent.kt [hunk 1]
```diff
@@ -201,6 +204,10 @@ class CardComponent private constructor(
         return if (cardNumber.length < BIN_VALUE_LENGTH) cardNumber else cardNumber.substring(0..BIN_VALUE_LENGTH)
     }
 
+    private fun getLastFourDigitsFromCardNumber(cardNumber: String): String {
```
Comment (caiofaustino) [inline card/src/main/java/com/adyen/checkout/card/CardComponent.kt:207]: do you think this needs a separate method?

File: card/src/main/java/com/adyen/checkout/card/CardComponentState.java [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
     private final CardType mCardType;
     private final String mBinValue;
+    private final String mLastFourDigits;
```
Comment (caiofaustino) [inline card/src/main/java/com/adyen/checkout/card/CardComponentState.java:23]: I wonder what this class would look like in Kotlin 👀

File: card/src/main/java/com/adyen/checkout/card/CardComponent.kt (hunk 1 above)
Comment (jreij) [reply card/src/main/java/com/adyen/checkout/card/CardComponent.kt:207]: Hmm maybe not, i had more logic inside it first then i noticed it's not needed, will fix 👍

File: card/src/main/java/com/adyen/checkout/card/CardComponentState.java (hunk 2 above)
Comment (jreij) [reply card/src/main/java/com/adyen/checkout/card/CardComponentState.java]: Check it out!

**Review by caiofaustino — APPROVED** [review]

**Pull request merged**